package.json

```diff
@@ -43,16 +43,17 @@
     "@emotion/react": "^11.7.1",
     "@emotion/styled": "^11.3.0",
     "@govtechsg/address-identity-resolver": "^1.4.1",
-    "@govtechsg/decentralized-renderer-react-components": "^3.6.2",
+    "@govtechsg/decentralized-renderer-react-components": "^3.8.0",
     "@govtechsg/ethers-contract-hook": "^2.2.0",
     "@govtechsg/oa-encryption": "^1.3.3",
     "@govtechsg/oa-verify": "^7.9.0",
-    "@govtechsg/open-attestation": "^6.3.0",
-    "@govtechsg/open-attestation-cli": "^1.40.4",
+    "@govtechsg/open-attestation": "^6.5.0",
+    "@govtechsg/open-attestation-cli": "^1.49.0",
     "@govtechsg/token-registry": "2.5.1",
-    "@govtechsg/tradetrust-ui-components": "^2.15.1",
-    "@govtechsg/tradetrust-utils": "^1.1.1",
+    "@govtechsg/tradetrust-ui-components": "^2.16.0",
+    "@govtechsg/tradetrust-utils": "^1.2.1",
```

Comment:
we should bump these separately next time

Reply:
yup, understand, will have a separate deps upgrade branch in future.